Backends/PDF/clim-pdf.asd

```diff
@@ -0,0 +1,16 @@
+
+;;; CLIM-PostScript is not a backend in the normal sense.
+;;; It is an extension (Chap. 35.1 of the spec) and is an
+;;; "included" part of McCLIM. Hence the defsystem is here.
+;;; 
```

Comment:
either change a comment to match the current system or remove it whatsoever, since it doesn't apply to cl-pdf